pkg/connector/helpers.go

```diff
@@ -575,3 +577,39 @@ func getGroupProjectsPermission(ctx context.Context, cli *client.DataCenterClien
 
 	return listGroup[groupPos].Permission, groupPos, err
 }
+
+func getError(err error) error {
+	var bitbucketErr *client.BitbucketError
+	if err == nil {
+		return nil
+	}
+
+	switch {
+	case errors.As(err, &bitbucketErr):
+		if bitbucketErr.ErrorCode == http.StatusUnauthorized {
+			return fmt.Errorf("%s", bitbucketErr.Error())
+		}
+	default:
+		return err
+	}
+
+	return nil
```

Comment:
We probably want to return the error here. In fact I think the switch can be omitted and you can just have an if statement that checks the error type. Something like:

```golang
if errors.As(err, &bitbucketErr) {
   return fmt.Errorf("%s %s", bitbucketErr.Error(), bitbucketErr.ErrorSummary)
}
return err
```

Reply:
Fixed